index.html

```diff
@@ -0,0 +1,131 @@
+            <h3 class="dog-name">Jack</h3>
+            <p class="dog-adoption-cost"><b>Cost to adopt:</b> $100</p>
+            <p class="dog-description">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a class="adopt-button" onclick="addToTotal(100)">Adopt</a>
+          </article>
+          <article class="dog-adoption-card">
+            <img class="dog-card-image" src="./images/jake-card.jpg" alt="" onclick="alertDogInfo('Jake', 'Golden Lab', '$100')">
+            <h3 class="dog-name">Jake</h3>
+            <p class="dog-adoption-cost"><b>Cost to adopt:</b> $100</p>
+            <p class="dog-description">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a class="adopt-button" onclick="addToTotal(100)">Adopt</a>
+          </article>
+          <article class="dog-adoption-card">
+            <img class="dog-card-image" src="./images/lucas-card.jpg" alt="" onclick="alertDogInfo('Lucas', 'German Shepherd Mix', '$100')">
+            <h3 class="dog-name">Lucas</h3>
+            <p class="dog-adoption-cost"><b>Cost to adopt:</b> $100</p>
+            <p class="dog-description">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a class="adopt-button" onclick="addToTotal(100)">Adopt</a>
+          </article>
+          <article class="dog-adoption-card">
+            <img class="dog-card-image" src="./images/luna-card.jpg" alt="" onclick="alertDogInfo('Luna', 'Golden Lab', '$100')">
+            <h3 class="dog-name">Luna</h3>
+            <p class="dog-adoption-cost"><b>Cost to adopt:</b> $100</p>
+            <p class="dog-description">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a class="adopt-button" onclick="addToTotal(100)">Adopt</a>
+          </article>
+          <article class="dog-adoption-card">
+            <img class="dog-card-image" src="./images/maximus-card.jpg" alt="" onclick="alertDogInfo('Maximus', 'Schnauzer Mix', '$100')">
+            <h3 class="dog-name">Maximus</h3>
+            <p class="dog-adoption-cost"><b>Cost to adopt:</b> $100</p>
+            <p class="dog-description">Corrum volorit iandae nimaxim cum restia volor reicid ut et etur sunt arum rendae pla endis re ea erum, qui doluptae</p>
+            <a class="adopt-button" onclick="addToTotal(100)">Adopt</a>
+          </article>
+        </section>
+        <a class="show-more-button" href="dogs.html">Show more</a>
+      </div>
+      <aside class="featured-blog-container">
+        <h2>From our blog</h2>
+        <article class="featured-blog-entry">
+          <div class="featured-blog-image">
+            <img src="./images/blog-1.jpg" alt="">
```

Comment:
Because these images are not responsive you are experiencing some issues with the scaling of the page at smaller resolutions.

Reply:
Thanks for the heads up. I was actually considering the best way to scale the blog images in particular, and am interested in your opinion. Would you recommend changing blog image size based on viewport width? Something else?

Reply:
I made style adjustments to margin and image width in the commit below to address scaling issues on smaller screens.